webknossos/webknossos/_nml/parameters.py

```diff
@@ -112,29 +126,43 @@ def _dump(self, xf: XmlWriter) -> None:
         xf.endTag()  # parameters
 
     @classmethod
-    def _parse_bounding_box(cls, bounding_box_element: Element) -> IntVector6:
-        return (
+    def _parse_bounding_box(cls, bounding_box_element: Element) -> BoundingBox:
+
+        topleft = (
             int(bounding_box_element.get("topLeftX", 0)),
             int(bounding_box_element.get("topLeftY", 0)),
             int(bounding_box_element.get("topLeftZ", 0)),
+        )
+        size = (
             int(bounding_box_element.get("width", 0)),
             int(bounding_box_element.get("height", 0)),
             int(bounding_box_element.get("depth", 0)),
         )
+        color = (
+            float(bounding_box_element.get("color.r", 0)),
+            float(bounding_box_element.get("color.g", 0)),
+            float(bounding_box_element.get("color.b", 0)),
+            float(bounding_box_element.get("color.a", 0)),
+        )
```

Comment:
I think the default should rather be `None` for the color, similar how the parsing is done for `Tree`.